translations/README.md

```diff
@@ -1,10 +1,10 @@
 # Hasura GraphQL Engine - Translations
 
-| File            | Language      |
-|-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md) |
-| Event Triggers  | [:uk: English](../event-triggers.md)  ￨ [:fr: Français](event-triggers.french.md)  |
-| Remote Schemas  | [:uk: English](../remote-schemas.md)  ￨ [:fr: Français](remote-schemas.french.md)  |
-| Contributing    | [:uk: English](../CONTRIBUTING.md)    ￨ [:fr: Français](CONTRIBUTING.french.md)    |
-| Security        | [:uk: English](../SECURITY.md)        ￨ [:fr: Français](SECURITY.french.md)        |
-| Code of Conduct | [:uk: English](../code-of-conduct.md) ￨ [:fr: Français](code-of-conduct.french.md) |
+| File            | Language                                                                                                                                                            |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Why do we need these long lines? The table structure as it was before was quite nice 🙂